pkg/api/customization/cluster/actionhandler.go

```diff
@@ -81,6 +100,14 @@ func (a ActionHandler) ClusterActionHandler(actionName string, action *types.Act
 		return a.RotateCertificates(actionName, action, apiContext)
 	case v3.ClusterActionRunCISScan:
 		return a.runCISScan(actionName, action, apiContext)
+	case v3.ClusterActionSaveAsTemplate:
+		if !canUpdateCluster() {
+			return httperror.NewAPIError(httperror.PermissionDenied, "can not save as an RKETemplate")
```

Comment:
nit: do you think we rephrase the error to "can not save cluster as an RKE template"?